docs/siem/detections/api/lists-index/lists-index-api-overview.asciidoc

```diff
@@ -0,0 +1,122 @@
+[[lists-index-api-overview]]
+== Lists index API
+
+Before using exception lists, use the `lists/index` API to create `.lists`
```

Comment:
Just a note - not really sure how/or if it's something you want to address, but I know that the terminology was one thing that really confused us as we were starting out. It might be helpful to somewhere explain that when referring to `exceptions` we are not denoting `exclusion` or `inclusion`, that part will be determined based on how they build out their exceptions.

Reply:
Thanks - I've updated the structure and text a bit to try and make things clearer.